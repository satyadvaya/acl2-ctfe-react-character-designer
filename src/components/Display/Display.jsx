export default function Display({ library }) {
  return (
    <div>
      {library.map((phrase) => (
        <p>{phrase}</p>
      ))}
    </div>
  );
}
