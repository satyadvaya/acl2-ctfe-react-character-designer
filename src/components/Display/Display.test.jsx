import { screen, render } from "@testing-library/react";
import Display from "./Display";

it("should render the Display component", () => {
  const { container } = render(
    <Display library={["hurrumph!", "riiilly?", "ahhyooogaah!"]} />
  );

  expect(container).toMatchSnapshot();
});
