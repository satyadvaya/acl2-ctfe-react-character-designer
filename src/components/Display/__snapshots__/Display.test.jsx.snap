// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Display component 1`] = `
<div>
  <div>
    <p>
      hurrumph!
    </p>
    <p>
      riiilly?
    </p>
    <p>
      ahhyooogaah!
    </p>
  </div>
</div>
`;
